less/notifications.less

```diff
@@ -0,0 +1,33 @@
+.id7-notifications-badge {
+  font-size: 0.8em;
+  animation-name: bounce;
```

Comment:
iIt fades in on load (that's done with jQuery). Do you want Animate.css's `pulse` animation here instead of `bounce`?